SPECS/grpc/grpc.spec

```diff
@@ -78,16 +78,30 @@ find %{buildroot} -name '*.cmake' -delete
 %{_datadir}/grpc/roots.pem
 
 %files devel
-%{_includedir}/*
```

Comment:
We should also remove `/usr/lib/libaddress_sorting.so.14*` and `/usr/lib/libupb.so.14` from the default packages as well, I guess.

Reply:
I actually will removed this from both since it is a third party file